.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
I'm not familiar with `.editorconfig`. Can you quickly teach me what value this adds?